File: Programmation/11_09_15/2.1.pl
#!/usr/bin/perl -w

$age=21;
$prenom="Marie" ;

# Affichage avec des guillemets doubles
$s1 = "$prenom a $age ans\n" ;
print "\$s1 : $s1" ;

#Affichage avec des guillemets simples
$s2 = '$prenom a $age ans\n' ;
print "\$s2 : $s2" ;

#Concaténation
$s3 = $prenom . " a " . $age . " ans\n" ;
print "\n\$s3 : $s3" ;

# exercise
$exo = "exo : " . $prenom . " aura " . ($age + 1) . " ans la semaine prochaine\n" ;
print $exo ;

File: Programmation/11_09_15/2.1.py
#!/usr/bin/python

age = 21
prenom = "Marie"

# Affichage avec des guillemets doubles
s1 = "s1 : %s a %s ans" % (prenom,str(age))
print s1

#Affichage avec des guillemets simples
s2 = 's2 : %s a %s ans' % (prenom,str(age))
print s2

#Concatenation
s3 = "s3 : " + prenom + " a " + str(age) + " ans"
print s3

# exercise
exo = "exo : " + prenom + " aura " + str(age+1) + " ans la semaine prochaine"
print exo


